AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -424,29 +425,88 @@ private async Task UpdateMultiDayHearing(EditMultiDayHearingRequest request, Gui
                 hearingsToUpdate.AddRange(futureHearings);
             }
 
-            foreach (var hearing in hearingsToUpdate)
+            if (_featureToggles.UseV2Api())
```

Comment:
Do you reckon we could break up UpdateMultiDayHearing() into a few more methods to reduce the complexity? Perhaps create one method for the V2 logic and one for the else block

Reply:
that's a good idea, updated